services/brig/src/Brig/API.hs

```diff
@@ -954,17 +954,16 @@ sitemap o = do
 
 setProperty :: UserId ::: ConnId ::: PropertyKey ::: JsonRequest PropertyValue -> Handler Response
 setProperty (u ::: c ::: k ::: req) = do
-    unless (Text.compareLength (Ascii.toText (propertyKeyName k)) maxKeyLen <= EQ) $
+    maxKeyLen <- fromMaybe 256 <$> view (settings . propertyMaxKeyLen)
+    maxValueLen <- fromMaybe 512 <$> view (settings . propertyMaxValueLen)
+    unless (Text.compareLength (Ascii.toText (propertyKeyName k)) (fromIntegral maxKeyLen) <= EQ) $
         throwStd propertyKeyTooLarge
     lbs <- Lazy.take (maxValueLen + 1) <$> liftIO (lazyRequestBody (fromJsonRequest req))
     unless (Lazy.length lbs <= maxValueLen) $
         throwStd propertyValueTooLarge
     val <- hoistEither $ fmapL (StdError . badRequest . pack) (eitherDecode lbs)
     API.setProperty u c k val !>> propDataError
     return empty
-  where
-    maxKeyLen   = 256
-    maxValueLen = 512
```

Comment:
Can you leave these lines in and rename them do `default...`?  Instead of inlining the default values above?